python/ray/util/sgd/v2/actor_group.py

```diff
@@ -0,0 +1,6 @@
+class ActorGroup:
+    pass
+
+
+class DistributedActor:
```

Comment:
What does DistributedActor do? Is this more like 'Worker'?

Reply:
would it be better to rename this to `BaseActor`?

Reply:
Will rename to `WorkerGroup` and `BaseWorker`.